app/src/main/java/com/antosik/benchproject/app/movie/details/viewModel/MovieDetailsViewModel.kt

```diff
@@ -30,8 +29,8 @@ class MovieDetailsViewModel @Inject constructor(
         get() = _isVisibleScreenElements
 
     private val _responseMovieDetails = MutableLiveData<MovieDetails>()
-    private val _isVisibleScreenElements = MutableLiveData(false)
-    private val _isLoaderVisible = MutableLiveData(true)
+    private val _isVisibleScreenElements = MutableLiveData<Boolean>()
```

Comment:
Why? Is there a reason to have it null at the beginning?

Reply:
I had a reason why but I don't remember it 😄 and now I tested with a default value and it works correctly. Therefore I deleted the changes to it